invenio_drafts_resources/records/api.py

```diff
@@ -227,3 +228,16 @@ def edit(cls, record):
                 fork_version_id=record.revision_id,
             )
         return draft
+
+    @classmethod
+    def cleanup_drafts(cls, td):
+        """Clean all the soft deleted drafts.
+
+        The drafts in the last timedelta spawn of time won't be soft deleted.
```

Comment:
small typo:
```suggestion
        The drafts in the last timedelta span of time won't be soft deleted.
```

Also, they "wont be soft deleted" but they are being "hard deleted", right? Im a bit confused with the wording 😅